Avoids
/data/data/com.termux/files/usr/bin/install: cannot stat 'html/*.png': No such file or directory

--- a/docs/doxygen/Makefile.am
+++ b/docs/doxygen/Makefile.am
@@ -7,7 +7,6 @@ if WITH_LIBWPD_DOCS
 install-data-am:
 	mkdir -p $(DESTDIR)$(docdir)/html
 	$(INSTALL_DATA) html/*.html $(DESTDIR)$(docdir)/html/
-	$(INSTALL_DATA) html/*.png $(DESTDIR)$(docdir)/html/
 	$(INSTALL_DATA) html/*.css $(DESTDIR)$(docdir)/html/
 
 uninstall-am:
